### DISCUSSION TITLE
Tree structure for lists?

### DISCUSSION THREAD

**Discussion by pschneider1968** [community/community#42811] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Lists are a great tool! But I think they could even be more useful if it was possible to put them in a tree structure, like a directory tree. After all, they are just containers for things, so we should be able to nest them.

I would like to suggest this idea as a feature improvement.

**Comment by mahmow55** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Lists are a great tool! But I think they could even be more useful if it was possible to put them in a tree structure, like a directory tree. After all, they are just containers for things, so we should be able to nest them.
> 
> I would like to suggest this idea as a feature improvement.

**Reply by mahmow55** [reply]
Yes

**Comment by jfullstackdev** [comment]
what if the Lists are not connected to one another ?

**Comment by pschneider1968** [comment]
Then they just stay root entries, as they are now. My suggestion was that you CAN nest entries, but you don't have to, of course!

**Comment by mehdiGihub** [comment]
بيع وشراء المنازل
